mapswipe_workers/mapswipe_workers/project_types/base/project.py

```diff
@@ -224,7 +224,7 @@ def save_to_postgres(self, project, groups, groupsOfTasks):
             VALUES (
               %s  -- created
               ,%s  -- createdBy
-              ,ST_Force2D(ST_Multi(ST_GeomFromText(%s, 4326)))  -- geometry
+              ,ST_Force2D(ST_GeomFromText(%s, 4326)) -- geometry
```

Comment:
this has an impact on the data type of the table in postgres, we need to alter the existing tables to accept any geometry and not only multipolygons.